mcmc/MCMCProcessor.h

```diff
@@ -14,6 +14,7 @@
 
 // ROOT includes
 #include "TObjArray.h"
+#include "TObjString.h"
```

Comment:
Do we need this extra include? It doesn't look like you've added in any dependency on TObjString

Reply:
So I only added this since when building with newer root versions TObjString.h has been removed from the other headers and has to be included separately.